plugins/rtmp-services/data/services.json

```diff
@@ -779,6 +779,16 @@
                 "bframes": 0
             }
         },
+        {
+          "name": "GameTips.TV - Service Stream Game Iran",
```

Comment:
Does the name need to be this long? I would think that `GameTips.TV` should be sufficient.

Reply:
Give it a change: GameTips.TV